ovh/resource_dedicated_server_networking_test.go

```diff
@@ -0,0 +1,94 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccresourceDedicatedServerNetworking(t *testing.T) {
+	confTemplates := []string{
+		testAccDedicatedServerNetworkingPublicVrack,
+		testAccDedicatedServerNetworkingSingleVrack,
+		testAccDedicatedServerNetworkingDualVrack,
+	}
+
+	for _, confTemplate := range confTemplates {
+		resource.Test(t, resource.TestCase{
+			PreCheck: func() {
+				testAccPreCheckCredentials(t)
+				testAccPreCheckDedicatedServer(t)
+			},
+			Providers: testAccProviders,
+			Steps: []resource.TestStep{
+				{
+					Config: testAccDedicatedServerNetworkingConfig(confTemplate),
+					Check: resource.ComposeTestCheckFunc(
+						resource.TestCheckResourceAttr(
+							"ovh_dedicated_server_networking.server", "status", "active"),
+					),
+				},
+			},
+		})
+	}
+}
+
+func testAccDedicatedServerNetworkingConfig(confTemplate string) string {
+	dedicated_server := os.Getenv("OVH_DEDICATED_SERVER")
```

Comment:
Others environment variables for testing purpose have this kind of format:
`OVH_CLOUD_PROJECT_KUBE_REGION_TEST`

Should start with `OVH_` and end with an `_TEST` please, like this:

`OVH_DEDICATED_SERVER_TEST`

Moreover, please add it in the Doc HomePage: https://registry.terraform.io/providers/ovh/ovh/latest/docs#testing-and-development

Thanks :-)